modules/swagger-codegen/src/main/resources/Javascript/api.mustache

```
@@ -80,11 +90,20 @@
       var accepts = [<#produces>'<& mediaType>'<#hasMore>, </hasMore></produces>];
       var returnType = <#returnType><&returnType></returnType><^returnType>null</returnType>;
 
-      return this.apiClient.callApi(
+      const promise = this.apiClient.callApi(
```

should we fail hard if someone uses `usePromises`?